__tests__/components/editor/ResourceTemplateForm.test.js

```diff
@@ -76,10 +76,6 @@ describe('<ResourceTemplateForm /> after fetching data from sinopia server', ()
 
       const wrapper = shallow(<ResourceTemplateForm.WrappedComponent {...rtProps} />)
       expect(wrapper.find('div.ResourceTemplateForm PropertyComponent').length).toEqual(1)
-      const inputType = wrapper.find('PropertyComponent').dive()
-
-      expect(inputType.find('Connect(InputLookupQA)').length).toEqual(1)
-      expect(inputType.find('Connect(InputListLOC)').length).toEqual(0)
```

Comment:
getting rid of these lines because ???

I feel like we are really shy on tests that check component interaction ... but I dunno if this is a great way to go.

Reply:
Because the ResourceTemplateForm test should be about the resource template form.  The PropertyComponent test is handling these cases as are *all* of the integration tests.  Furthermore, since the loading is dynamic now, enzyme won't be able to see these components as they're actually nested in a Suspense/lazy load.